### PR TITLE
Add previous direction to Player type

### LINKED ISSUE #129: Preserve player direction even if player stops
Currently, we determine `Player.Direction` by the player velocity, and if the velocity is zero, then it defaults to `Right`.

We should preserve the last direction when the player stops.

See `TODO[#xxx]` in the code.

### PR DESCRIPTION
**Pull request by Fantoom**
Closes #129 

I am not sure is it a good way

### CODE REVIEW

**Comment by Fantoom** [comment]
Also, I have added sprite rotation based on direction

File: O21.Game/Engine/Entities.fs [hunk 1]
```diff
@@ -3,17 +3,20 @@ namespace O21.Game.Engine
 type Player = {
     TopLeft: Point
     Velocity: Vector
+    PrevDirection: HorizontalDirection
```
Comment (gsomix) [inline O21.Game/Engine/Entities.fs:6]: Probably we need to remove computed property `Direction` and just use record's field named `Direction` instead. IIRC left direction is only possible when special bonus taken.

**Review by gsomix — CHANGES_REQUESTED** [review]

File: O21.Game/Engine/Entities.fs (hunk 1 above)
Comment (ForNeVeR) [reply O21.Game/Engine/Entities.fs:6]: Ha, I totally forgot about that.

Yeah, I'll file an issue for it, and refactor a bit here.

**Review by ForNeVeR — APPROVED** [review]
Thanks!

**Pull request merged**